addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_ubl_au.py

```diff
@@ -105,6 +102,10 @@ def test_export_import_invoice(self):
                 <xpath expr=".//*[local-name()='PaymentMeans']/*[local-name()='PaymentID']" position="replace">
                     <PaymentID>___ignore___</PaymentID>
                 </xpath>
+                <xpath expr=".//*[local-name()='AdditionalDocumentReference']/*[local-name()='Attachment']/*[local-name()='EmbeddedDocumentBinaryObject']" position="attributes">
+                    <attribute name="mimeCode">application/pdf</attribute>
+                    <attribute name="filename">{invoice.invoice_pdf_report_id.name}</attribute>
```

Comment:
I'd get a related field for that (maybe compute+inverse to manage the case where `invoice_pdf_report_id` is not set?)

```python
invoice_pdf_report_name = fields.Char(related='invoice_pdf_report_id.name', readonly=False)
```